containers/datalab/content/run.sh

```diff
@@ -206,20 +206,20 @@ mkdir -p /content/datalab/notebooks
 
 # Fetch docs and tutorials. This should not abort startup if it fails
 {
-if [ -d /content/datalab/docs ]; then
+if [ -d /content/datalab/ga ]; then
```

Comment:
The problem is indeed --single-branch. We need to add a command:
git config remote.origin.fetch "+refs/heads/ga:refs/remotes/origin/ga"
to be able to fetch and then switch to ga branch locally.